source/find_getmore_killcursors_commands.rst

```diff
@@ -731,6 +735,8 @@ More in depth information about passing read preferences to Mongos can be found
 
 Changes
 =======
+2021-12-14 Exhaust cursors may fallback to non-exhaust cursors on 5.1+ servers.
```

Comment:
In addition to permitting fallback behavior, may want to include note regarding revision of prior OP_MSG requirement on 3.6+ servers being relaxed to 5.1+.